docs/src/pages/NativePickerPage/NativePickerPage.js

```diff
@@ -29,8 +29,6 @@ const components = [
   {
     id: 'default',
     title: 'Default',
-    blurb:
-      'Picker trigger elements which display the currently selected value and can be tapped to present the actual picker.',
```

Comment:
This sentence didn't even make sense so I just removed it. It's self evident anyway.